mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -263,37 +245,20 @@ export const PasscodePrompt = React.memo((props: PasscodePromptProps): JSX.Eleme
   )
 })
 
-function SuccessMessage ({ message }: { message?: { title: string, description: string } }): JSX.Element | null {
-  const { isLight } = useThemeContext()
-  if (message === undefined) {
-    return null
-  }
-
+function SuccessIndicator (): JSX.Element {
   return (
-    <View style={tailwind('flex-col items-center p-6')}>
-      {isLight ? <TransactionSignedLight /> : <TransactionSignedDark />}
-      <ThemedText style={tailwind('text-center text-xl font-bold mt-5')}>
-        {message.title}
-      </ThemedText>
-
-      <ThemedText style={tailwind('text-sm text-center')}>
-        {message.description}
-      </ThemedText>
-    </View>
-  )
-}
-
-function Loading ({ message }: { message?: string }): JSX.Element | null {
-  if (message === undefined) {
-    return null
-  }
-  return (
-    <View style={tailwind('flex-row justify-center p-2')}>
-      <ThemedActivityIndicator />
-
-      <ThemedText style={tailwind('ml-2 text-sm')}>
-        {message}
-      </ThemedText>
+    <View style={tailwind('flex flex-col items-center py-4 my-1')}>
+      <View
+        style={tailwind('h-9 w-9 rounded-full flex justify-center items-center border bg-green-v2 border-green-v2')}
+      >
+        <ThemedIcon
+          size={36}
+          name='check'
```

Comment:
It works but there is a slight different from using the actual icon which includes the circular background
According to figma, it is `check-circle` from MaterialIcons 
<img width="240" alt="image" src="https://user-images.githubusercontent.com/40191153/178475360-65e76a07-b507-4477-b56f-bd6ac2105e3c.png">

Reply:
Checked and the suggested icon has a circular background rather than a full background like on figma. Should we style it to look like figma or keep the circular background?